bool body_lift
bool grasp
bool push
bool camera_lift
bool flip
